src/Runner.jl

```diff
@@ -499,7 +497,7 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
             "FC"     => "$(host_target)-f77",
             "LD"     => "$(host_target)-ld",
             "NM"     => "$(host_target)-nm",
-            "OBJC"   => "$(host_target)-objc",
+            "OBJC"   => "$(host_target)-cc",
```

Comment:
The host Objective-C compiler is _**required**_ by Meson (:facepalm: but I think in newer versions this should be fixed), but it isn't a big deal as this variable is set only inside the `meson` wrapper.